vmc/resource_vmc_sddc.go

```diff
@@ -364,8 +367,47 @@ func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 	sddcID := d.Id()
 	orgID := (m.(*ConnectorWrapper)).OrgID
 
+	// Convert sddc from 1node to default
+	toConvertSddc := false
+	if d.HasChange("sddc_type") {
+		oldTmp, newTmp := d.GetChange("sddc_type")
+		oldType := oldTmp.(string)
+		newType := newTmp.(string)
+
+		// Validate for convert type params
+		if oldType == "1NODE" && (newType == "" || newType == "DEFAULT") {
+			oldTmp, newTmp := d.GetChange("num_host")
+			oldNum := oldTmp.(int)
```

Comment:
you can directly get oldNum := d.GetChange("num_host").(int) Remove multiple variables oldNum, oldType, newNum, newType usage.

Reply:
Make these changes else where in the code if needed

Reply:
There is not a elegant way to convert multiple return value without temp variable